### PR TITLE
add item to troubleshooting article

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[54ad339](https://github.com/OfficeDev/office-js-docs-pr/pull/1136/commits/54ad339b9d499b56a6ac0c647a8580586ecbcb78)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/troubleshooting/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906252210315468-1136%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=T0m21pmDetETEJ4zptk%2FFnC0fUeM%2BYA4bmKDIipo8Hg%3D&st=2019-06-25T22%3A06%3A35Z&se=2019-07-26T22%3A11%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/testing-and-troubleshooting.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ The following table lists common error messages that users might encounter and s
 |Error: Object doesn't support property or method 'defineProperty'|Confirm that Internet Explorer is not running in Compatibility Mode. Go to Tools >  **Compatibility View Settings**.|
 |Sorry, we couldn't load the app because your browser version is not supported. Click here for a list of supported browser versions.|Make sure that the browser supports HTML5 local storage, or reset your Internet Explorer settings. For information about supported browsers, see [Requirements for running Office Add-ins](../concepts/requirements-for-running-office-add-ins.md).|
 
+## When installing an add-in, you see "Error loading add-in" in the status bar
+
+Close Office, verify that the manifest is valid, restart the add-in, and then try installing again. Also, give us feedback: if using Excel on Windows or Mac, you can report feedback to the Office extensibility team directly from Excel. To do this, select **File** | **Feedback** | **Send a Frown**. Sending a frown will provide the necessary logs to understand the issue you are hitting.
```
Comment (AlexJerabek) [inline docs/testing/testing-and-troubleshooting.md]: Three nits:
- Consider changing "Also, give us feedback" to "You can also give us feedback".
- Consider changing "will provide" to "provides".
- Consider condensing "understand the issue you are hitting" to "understand your issue".

Comment (AlexJerabek) [inline docs/testing/testing-and-troubleshooting.md]: Should the first sentence be four numbered bulletpoints instead? That seems to be the usual pattern in the article.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[975798e](https://github.com/OfficeDev/office-js-docs-pr/pull/1136/commits/975798e7d9cb71695c43c45ab5a387b6bf9d8c7f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/testing-and-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/troubleshooting/docs/testing/testing-and-troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907011941153136-1136%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gU2WKPy6RS25ruKhndxBFeFCM1MCX%2FEdJyznWT%2BjOEk%3D&st=2019-07-01T19%3A37%3A16Z&se=2019-08-01T19%3A42%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**